.github/workflows/verible-ci.yml

```diff
@@ -193,6 +193,7 @@ jobs:
     needs: [ Check, Build ]
     runs-on: ubuntu-20.04
     name: 📦 Release
+    if: ${{github.ref == 'refs/heads/main' || github.ref == 'refs/head/master'}}
```

Comment:
(still waiting for the CI to finish, but question:) Wouldn't this also need to be qualified that this only happens on `push` ?

Reply:
Don't wait up - it could easily be many hours (that's what prompted this endeavour 😄).

I'll add it for clarity, but`push` is implied. From [here](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context):

> `github.ref` - The branch or tag ref that triggered the workflow run. For branches this in the format refs/heads/<branch_name>, and for tags it is refs/tags/<tag_name>.

The only way for the ref to be main/master would be for main/master to _trigger_ the workflow. I guess _technically_ you could open a PR from main->main, but that seems like an odd thing to do 😄

Reply:
Done, although I did find, scrolling down a bit, a similar guard already existed on the _step_. I've removed that since it's redundant now.